pkg/sourcereader/embedded.go

```diff
@@ -246,13 +186,11 @@ func (r EmbeddedSourceReader) GetModule(modPath string, copyPath string) error {
 		return fmt.Errorf("Source is not valid: %s", modPath)
```

Comment:
For this function, you want to copy a directory from the embedded FS to disk. The path itself appears relative, so it would only be able to copy what is relative to this directory in this implementation. Instead, you'd want to do something similar to `copyFSToTempDir`, but instead of copying to a temp directory, copy to the `copyPath` directly.

In fact, I'd recommend keeping it as is and updating it in a follow-up PR. There is some additional safety builtin when copying to a temp directory first and using the same logic as the other sourcereader's (see the git.go for an example).